src/types/ErrorType.ts

```diff
@@ -0,0 +1,7 @@
+export enum ErrorType {
+  get = 'Unable get response from the server',
+  post = 'Unable to add a todo',
+  delete = 'Unable to delete a todo',
+  patch = 'Unable to update a todo',
+  success = '',
```

Comment:
It is not recommended to include a 'success' case in an ErrorType enum. Consider creating a separate enum for success messages or handling success separately from errors.